### ISSUE TITLE
tables referring to multiple regions

### ISSUE THREAD

**Issue by giovp** [scverse/spatialdata#99]
in both nanostring cosmx and xenium we available info on labels and labels' centroids. Therefore, the spatialdata object will (could) have 2 regions representing the same instance (with potentially same `instance_id`) and a single table annotating both. 
I believe we discussed this but I don't remember how we ended up resolving this issue. 

I think one option we discussed was that the centroids labels would just be stored in table.obsm["spatial"] essentially as an annotation on the labels (and useful for plotting etc) but without any constraint or specifications (e.g. no coordinate systems and/or transforms).  If that's the case, then there would be no `Shapes` region for the centroids.

Any idea on how to resolve this issue? 

related to https://github.com/scverse/spatialdata/issues/43

**Comment by LucaMarconato** [comment]
Hi, from the discussion we settled, as you mentioned, on storing the centroids in the table as a separate column, and if I recall correctly we said to store the centroids in the implicit coordinates. This is meant to be a way to unblock this now and not the definitive solutions, since if for instance the centroids come from a labels and they are cropped, then the implicit coordinates stored in the table would not be correct anymore. I also agree of removing the `Shapes` from storage, they could be created in-place at any time if needed.

**Comment by kevinyamauchi** [comment]
My recollection is the same as @LucaMarconato 's.

**Comment by LucaMarconato** [comment]
Closed as duplicate of #34 and https://github.com/scverse/spatialdata/issues/123.
